docs/IE_DG/protecting_model_guide.md

```diff
@@ -43,6 +43,10 @@ Extensions](https://software.intel.com/en-us/sgx).
 Use `InferenceEngine::Core::ReadNetwork()` to set model representations and
 weights respectively.
 
+Currently there are no possibility to read external weights from memory for ONNX models.
+Function `ReadNetwork(const std::string& model, const Blob::CPtr& weights)`
+should be called with `weights` equal `nullptr` in such a case.
```

Comment:
`nullptr` doesn't work. We need to use empty blob.

Reply:
done